scripts/ros_connect.sh

```diff
@@ -105,42 +102,42 @@ ros_connect() {
 	while [ "$#" -gt 0 ]; do
 		case $1 in
 			-h|--help)
-				echo "Usage: ros_connect [OPTION] [HOSTNAME]..."
+				echo "Usage: ros_connect [OPTION]..."
 				echo "Manager for connections to remote roscores."
 				echo ""
 				echo "Option		GNU long option		Meaning"
 				echo "-h		--help			Display the help menu"
-				echo "-p		--persistence		Toggle persistence across shells"
-				echo "-o		--one-time		Only set the roscore for this shell"
 				echo "-n [HOSTNAME]	--hostname		Manually pass in a hostname"
+				echo "-o		--one-time		Only set the roscore for this shell"
+				echo "-p		--persistence		Toggle persistence across shells"
 				HOST_DISCOVERY=false
 				PERSIST=false
 				shift 1
 				;;
-			-p|--persistence)
-				if [ -z `cat $PERSISTENCE_FILE | grep "disabled"` ]; then
-					echo "disabled" > $PERSISTENCE_FILE
+			-n|--hostname)
+				HOST="$2"
+				if [ "$HOST" != "localhost" ]; then
+					set_ros_ip
 				else
-					echo $DEFAULT_HOST > $PERSISTENCE_FILE
+					unset_ros_ip
 				fi
+				set_ros_master $HOST
 				HOST_DISCOVERY=false
-				PERSIST=false
-				shift 1
+				shift 2
 				;;
 			-o|--one-time)
 				PERSIST=false
 				shift 1
 				;;
-			-n|--hostname)
-				HOST="$2"
-				if [ "$HOST" != "localhost" ]; then
-					set_ros_ip
+			-p|--persistence)
+				if [ -z `cat $PERSISTENCE_FILE | grep "disabled"` ]; then
```

Comment:
You didn't provide a reason for why it would be a bad idea. But since we already have a ~/.milrc, it seems like a ~/.uf-mil/ could be a clean way to store all this stuff.